### PR TITLE
Show friendly message to install action mailbox if the related table does not exist

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
- This is similar to the work done in https://github.com/rails/rails/pull/31534

https://monosnap.com/file/Ped2VSD5gmjYxjp0xPzWR2BEs17Qbx

### CODE REVIEW

**Comment by kaspth** [comment]
Did you already have Active Storage enabled in the app you tested with? I’m not sure the error message would hit on Action Mailbox first?

**Comment by prathamesh-sonpatki** [comment]
@kaspth No I didn't have active storage enabled. I was testing the conductor page and clicked on the Action Mailbox Inbound emails link: https://monosnap.com/file/RKUdXiXgy5fY4oAR6fGb6ksle33yRG

File: actionpack/lib/action_dispatch/middleware/templates/rescues/invalid_statement.text.erb [hunk 1]
```diff
@@ -6,6 +6,8 @@
 <%= @exception.message %>
 <% if defined?(ActiveStorage) && @exception.message.match?(%r{#{ActiveStorage::Blob.table_name}|#{ActiveStorage::Attachment.table_name}}) %>
 To resolve this issue run: rails active_storage:install
+<% if defined?(ActionMailbox) && @exception.message.match?(%r{#{ActionMailbox::InboundEmail.table_name}}) %>
+  To resolve this issue run: rails action_mailbox:install
```
Comment (kaspth) [inline actionpack/lib/action_dispatch/middleware/templates/rescues/invalid_statement.text.erb]: Probably needs to be outdented like the line above.

**Comment by kaspth** [comment]
Cool! I just wanted to make sure we didn't say that "Active Storage must be installed" for Action Mailbox being missing (yes, it's a dependency but the error should focus on Action Mailbox).

**Comment by kaspth** [comment]
If you want to convert this to use #34788, do feel free to translate them! Perhaps as a PR to that branch?

cc @gsamokovarov

File: actionpack/lib/action_dispatch/middleware/templates/rescues/invalid_statement.text.erb (hunk 1 above)
Comment (prathamesh-sonpatki) [reply actionpack/lib/action_dispatch/middleware/templates/rescues/invalid_statement.text.erb]: Updated!

**Comment by prathamesh-sonpatki** [comment]
> If you want to convert this to use #34788, do feel free to translate them! Perhaps as a PR to that branch?
> 
> cc @gsamokovarov

@kaspth Sure, I will do that. Can I do that once this PR is merged or are you suggesting to convert this PR itself?

**Comment by kaspth** [comment]
I meant after, but it wasn't clear in the message. But that's fixed now that I merged 😄

**Pull request merged**